test/ruleset/rules/v2/asyncapi2-message-examples.spec.ts

```diff
@@ -378,19 +390,32 @@ testRule('asyncapi2-message-examples', [
                   {name: 'speed', type: 'string'}
                 ]
               },
+              headers: {
```

Comment:
Pretty sure headers cannot be avro 🤔 At least not in v2

Reply:
No I added the headers to the original tests (schemaFormat not set case) so that It added confidence that header behaviour was maintained across both versions of the rule (before we decided to split the rules)
I can revert this.

Though the new info that headers cannot be v2 means I can simplify the new v2 rule!